tests/integration/operator/switch_cr_test.go

```diff
@@ -255,6 +255,22 @@ func cleanupInClusterCRs(t framework.TestContext, cs cluster.Cluster) {
 	} else {
 		t.Logf("failed to list existing CR: %v", err.Error())
 	}
+
+	// wait for pods in istio-system to be deleted
+	err := retry.UntilSuccess(func() error {
+		podList, err := cs.Kube().CoreV1().Pods(IstioNamespace).List(context.TODO(), kubeApiMeta.ListOptions{})
```

Comment:
We maybe want to add a label selector to only match the appropriate `istio.io/rev`. But I guess if this passes tests it is ok

Reply:
This function deletes all operators no matter if istio.io/rev is set, so it should delete all pods, shouldn't it? Please correct me if i'm wrong

Reply:
I was thinking maybe there are non-istio pods that happen to be deployed in the users cluster. But probably not worth bothering with it